src/main/resources/db/demo/11_text.sql

```
@@ -0,0 +1,366 @@
+--   · [제목] 대괄호 제거 — 블록 제목과 중복이다
+--   · col_span 1 (297px) 이라 한 블록 150자 내외로 줄인다
+-- =====================================================================
+
+-- ── 입점 검토 ────────────────────────────────────────────────────────
+UPDATE `text` SET content='자사몰 매출이 3개월 연속 빠졌다.
+
+- 8월 4,200만
+- 9월 4,050만
+- 10월 **3,800만**
+
+신규 유입 채널이 필요하다. 후보 3사 중 카테고리 적합성과 트래픽 규모를 보고 무신사를 먼저 본다.' WHERE txt_id=9001;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)